types/oxon/buttonboard/meta.json

```diff
@@ -1,9 +1,9 @@
 {
-  "name": "Buttonboard (LoRaWAN)",
+  "name": "Buttonboard",
```

Comment:
changes in name will need to be done manually, therefore coordination when merging this is needed